tests/functional/metadata/mongodb/putObject.spec.js

```diff
@@ -0,0 +1,442 @@
+const async = require('async');
```

Comment:
we have no such convention of having `.spec` extension for test files : so we should stick with the current approach

Reply:
this was introduced here https://github.com/scality/Arsenal/pull/1723

Reply:
fair enough :-)